dotcom-rendering/src/layouts/StandardLayout.tsx

```diff
@@ -417,6 +435,8 @@ export const StandardLayout = (props: WebProps | AppProps) => {
 
 	const renderAds = isWeb && canRenderAds(article);
 
+	const podcastSeries = article.tags.find((tag) => tag.type === 'Series');
```

Comment:
Minor - I think this is telling us the article is of series type and not whether it is a podcast series?

```suggestion
	const isSeries = article.tags.find((tag) => tag.type === 'Series');
```